components/tag-list/tag-list.js

```diff
@@ -179,13 +180,12 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 			<div role="application" class="tag-list-outer-container" style="${styleMap(outerContainerStyles)}">
 				<d2l-button-subtle aria-hidden="true" slim text="${this.localize('components.tag-list.num-hidden', { count: '##' })}" class="d2l-tag-list-hidden-button"></d2l-button-subtle>
 				${this.arrowKeysContainer(list)}
-				<div id="d2l-tag-list-description" hidden>${this.description}</div>
 			</div>
 		`;
 	}
 
 	async arrowKeysFocusablesProvider() {
-		return this._showHiddenTags ? this._items : this._items.slice(0, this._chompIndex);
+		return this._getVisibleEffectiveChildren();
```

Comment:
I changed the keyboard arrow key work flow to include the clear and show more/less buttons. In my opinion, when using a screenreader this made the workflow more intuitive (e.g., arrow key onto the "+ X more" button, the user immediately knows there are more hidden tags rather than not knowing that until they tab onto that button)